### PR TITLE
✨ Add _NCS#triggerClientEvent

### PR DESCRIPTION
**Pull request by Walker974**

### CODE REVIEW

File: src/core/server.lua [hunk 1]
```diff
@@ -1 +1,9 @@
-_NCS:trace("Loading NCS Core, please wait...")
\ No newline at end of file
+_NCS:trace("Loading NCS Core, please wait...")
+
+---triggerClientEvent
+---@param eventName string
+---@return void
+---@public
+function _NCS:triggerClientEvent(eventName, ...)
+    TriggerClientEvent(self:formatEvent(eventName), ...)
```
Comment (PABLO-1610) [inline src/core/server.lua:8]: Missing the target ID, please refer to : 

`TriggerClientEvent(string eventName, int playerId[, ...])`

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]

File: src/core/server.lua (hunk 1 above)
Comment (Walker974) [reply src/core/server.lua]: is it correct now?

File: src/core/server.lua [hunk 2]
```diff
@@ -5,5 +5,6 @@ _NCS:trace("Loading NCS Core, please wait...")
 ---@return void
 ---@public
 function _NCS:triggerClientEvent(eventName, ...)
-    TriggerClientEvent(self:formatEvent(eventName), ...)
+    TriggerClientEvent(string eventName, int playerId[, ...])
```
Comment (PABLO-1610) [inline src/core/server.lua:8]: No, you need to add the targetId in the func and in the triggerClientEvent.

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]

**Comment by Walker974** [comment]
thank you for your help.

**Pull request merged**